gcc/config/arc/arc.opt

```diff
@@ -102,6 +102,25 @@ Same as -mcpu=archs
 ; END backward-compatibility aliases, translated by DRIVER_SELF_SPECS
 
 TargetVariable
+unsigned int arc_jli_func_addr = 0
+
+mjli-func-addr=
+Target RejectNegative Joined Var(arc_jli_func_addr) Enum(arc_jli_func_addr_type) Init(ARC_FUNC_ADDR_COMPAT)
+Specify the way the address of a function in the JLI table is obtained. Option 'compat' will use the Metaware method (jli_call_always will use the address of the function and jli_call_dynamic will use the address of the JLI entry). Option 'init' will use the address of the JLI entry using the jli_base register when the function pointer is saved. Option 'always' will use a stub function to call the JLI entry relative to the jli_base register upon calling the function pointed to.
```

Comment:
Here should be a short description. The long description should go into backend documentation file.